src/api-service/__app__/onefuzzlib/tasks/main.py

```diff
@@ -140,6 +141,13 @@ def search_expired(cls) -> List["Task"]:
             query={"state": TaskState.available()}, raw_unchecked_filter=time_filter
         )
 
+    @classmethod
+    def try_get_by_task_id(cls, task_id: UUID) -> Optional["Task"]:
```

Comment:
Do we need try_get_by_task_id and get_by_task_id, where one returns Union[Null, Task], and the other returns Union[Error, Task]?